code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -513,6 +513,8 @@
 
 	//use power
 	if(!use_energy(active_power_usage))
```

Comment:
Make sure the force parameter is false too.

Reply:
Done